packages/vats/src/core/chain-behaviors.js

```diff
@@ -360,28 +360,68 @@ harden(makeBridgeManager);
  */
 export const makeChainStorage = async ({
   consume: { loadCriticalVat, bridgeManager: bridgeManagerP },
-  produce: { chainStorage: chainStorageP },
+  produce: {
+    chainStorage: chainStorageP,
+    storageBridgeManager: storageBridgeManagerP,
+  },
 }) => {
   const bridgeManager = await bridgeManagerP;
   if (!bridgeManager) {
     console.warn('Cannot support chainStorage without an actual chain.');
     chainStorageP.resolve(null);
+    storageBridgeManagerP.resolve(null);
     return;
   }
 
-  const ROOT_PATH = STORAGE_PATH.CUSTOM;
-
   const storageBridgeManager = E(bridgeManager).register(BRIDGE_ID.STORAGE);
+  storageBridgeManagerP.resolve(storageBridgeManager);
 
   const vat = E(loadCriticalVat)('bridge');
   const rootNodeP = E(vat).makeBridgedChainStorageRoot(
     storageBridgeManager,
-    ROOT_PATH,
+    STORAGE_PATH.CUSTOM,
     { sequence: true },
   );
   chainStorageP.resolve(rootNodeP);
 };
 
+/**
+ * @param {BootstrapSpace & {
+ *   consume: { loadCriticalVat: ERef<VatLoader<ChainStorageVat>> }
+ * }} powers
+ */
+export const produceHighPrioritySendersManager = async ({
+  consume: { loadCriticalVat, storageBridgeManager: storageBridgeManagerP },
+  produce: { highPrioritySendersManager: managerP },
+}) => {
+  const storageBridgeManager = await storageBridgeManagerP;
+  if (!storageBridgeManager) {
+    console.warn(
+      'Cannot provide highPrioritySendersManager without an actual chain.',
+    );
+    managerP.resolve(null);
+    return;
+  }
+
+  const sendersNode = E(
+    E(loadCriticalVat)('bridge'),
+  ).makeBridgedChainStorageRoot(
+    storageBridgeManager,
+    STORAGE_PATH.HIGH_PRIORITY_SENDERS,
+    { sequence: false },
+  );
+
+  // NB: this is a non-durable Far object. If the bootstrap vat (where this object is made)
+  // were to be terminated, the object references to this would be severed.
+  // When bootstrap vat is restarted by bootstrap config containing the state
+  // extracted from IAVL but the contracts holding this manager will need to be
+  // updated with the new object. That can be done without a contract upgrade by
+  // restarting the contracts with the new object in privateArgs.
```

Comment:
After discussion, the plan at https://github.com/Agoric/agoric-sdk/issues/4548 to move the powers to a new vat using core eval would allow performing a phased transition without temporary breakage.

Reply:
Documented within repo with https://github.com/Agoric/agoric-sdk/pull/7500/commits/99f10d97b724fe502d31d543d36663982ad5b08f